libs/ledger/src/consensus/simulated_pow_consensus.cpp

```diff
@@ -123,6 +123,10 @@ NextBlockPtr SimulatedPowConsensus::GenerateNextBlock()
   FETCH_LOG_DEBUG(LOGGING_NAME, "Generating block. Current time: ", current_time_ms,
                   " deadline: ", decided_next_timestamp_ms_);
 
+  // Round up to the nearest second for block timestamp
+  auto current_time_s_ceiled =
```

Comment:
Looks like it is not changed below, so m.b. "auto const" ?

Reply:
m.b = maybe?
Sure, I have made the change now 👍